From d738328ecf8708c2a19fa5ded415c8e87832634d Mon Sep 17 00:00:00 2001
From: Wilken Gottwalt <wilken.gottwalt@posteo.net>
Date: Mon, 26 Aug 2024 09:05:41 +0200
Subject: [PATCH] libgccjit: Remove obsolete texinfo statements
To: gcc-patches@gcc.gnu.org
Cc: David Malcolm <dmalcolm@redhat.com>

Remove texinfo statements which are obsolete for a while now.

libgccjit.texi:18: warning: @definfoenclose is obsolete
libgccjit.texi:19: warning: @definfoenclose is obsolete

gcc/jit:
	* docs/_build/texinfo/libgccjit.texi: Remove obsolete texinfo statements.

Signed-off-by: Wilken Gottwalt <wilken.gottwalt@posteo.net>
---
 gcc/jit/docs/_build/texinfo/libgccjit.texi | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/gcc/jit/docs/_build/texinfo/libgccjit.texi b/gcc/jit/docs/_build/texinfo/libgccjit.texi
index a69efeaa4a8..e7abc49b160 100644
--- a/gcc/jit/docs/_build/texinfo/libgccjit.texi
+++ b/gcc/jit/docs/_build/texinfo/libgccjit.texi
@@ -14,9 +14,6 @@
 @direntry
 * libgccjit: (libgccjit.info). GCC-based Just In Time compiler library.
 @end direntry
-
-@definfoenclose strong,`,'
-@definfoenclose emph,`,'
 @c %**end of header
 
 @copying
-- 
2.46.0

